civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -375,6 +376,8 @@ function _rebuild_length_of_service_view() {
 function _rebuild_absence_activity_view() {
   $civi_settings = parse_url(CIVICRM_DSN);
   $civi_db_name = trim($civi_settings['path'], '/');
+  $calculationUnitOptions = array_flip(AbsenceType::buildOptions('calculation_unit', 'validate'));
+  $hoursUnit = $calculationUnitOptions['hours'];
```

Comment:
Code looks fine, just a bit confused here. What are the possible values for `$hoursUnit`?

Reply:
The calculation unit option group values are 1 and 2. `1` for Absence types having leave request  in days. `2` for absence type having leave request in hours.
In this case above `$hoursUnit = 2`

Reply:
I see, so it's just fetching the option value `value` and using it in the view? That makes sense

Reply:
It's a minor issue, but I think since in this case `$hoursUnit = 2` doesn't really make sense, it might be better to name this variable `$hoursUnitID` or `$hoursUnitCode`. The calculation units are hours and days, so 2 is the identifier for the hours unit. Like I said, it's minor though, so I'll leave it up to you to decide.